.github/workflows/auth-beta-release.yml

```diff
@@ -332,7 +332,7 @@ jobs:
                   path: mobile/apps/auth/artifacts/*
 
     build-macos:
-        runs-on: macos-13 # latest is 12
+        runs-on: macos-latest
```

Comment:
Instead of latest, can we pin specific version to avoid any surprise related to builds?